tools/setup_environment.py

```diff
@@ -55,8 +73,8 @@ def main(user):
     log_dir = "/var/log/host-os"
     repo_dir = "/var/lib/host-os/repositories"
```

Comment:
Maybe use `/var/lib/host-os` instead so that we don't need to add each directory individually should we decide to put more stuff under host-os. I remember we discussed about moving the 'versions' clone to `/var/lib/host-os` too.